README.md

```diff
@@ -20,6 +20,43 @@ A fully working use-case and example of library usage is available in the
 There is a one-time setup phase to prime the code for autometrics. Once this
 phase is accomplished, only calling `go generate` is necessary.
 
+### Install the go generator.
+
+The generator is the binary in cmd/autometrics, so the easiest way to get it is
+to install it through go:
+
+```console
+go install github.com/autometrics-dev/autometrics-go/cmd/autometrics@latest
+```
+
+### Import the libraries and initialize the metrics
+
+In the main entrypoint of your program, you need to both add package
+
+``` go
+import (
+	am "github.com/autometrics-dev/autometrics-go/pkg/autometrics/prometheus"
+)
+```
+
+And then in your main function initialize the metrics
+
+``` go
+am.Init(nil, am.DefBuckets)
+```
+
+**_Warning_**: if you want to enable alerting from Autometrics, you **MUST**
+have the `--latency-ms` values to match the values given in your buckets. The
+values in the buckets are given in _seconds_. By default, the generator will
+error and tell you the valid default values if they don't match.
+If the default values do not match you use case, you can change the buckets in
+the init call, and add a `-custom-latency` argument to the `//go:generate` invocation.
```

Comment:
we can make thsi more prominent using a [special gfm feature](https://github.com/orgs/community/discussions/16925):

```suggestion
> **Warning**
> If you want to enable alerting from Autometrics, you **MUST**
have the `--latency-ms` values to match the values given in your buckets. The
values in the buckets are given in _seconds_. By default, the generator will
error and tell you the valid default values if they don't match.
If the default values do not match you use case, you can change the buckets in
the init call, and add a `-custom-latency` argument to the `//go:generate` invocation.
```

it will look like this: 

> **Warning**
> If you want to enable alerting from Autometrics, you **MUST**
have the `--latency-ms` values to match the values given in your buckets. The
values in the buckets are given in _seconds_. By default, the generator will
error and tell you the valid default values if they don't match.
If the default values do not match you use case, you can change the buckets in
the init call, and add a `-custom-latency` argument to the `//go:generate` invocation.

Reply:
Oh that's perfect, thanks for that!